packages/react-core/src/next/components/Select/examples/SelectTypeahead.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { Select, SelectOption, SelectList, SelectOptionProps } from '@patternfly/react-core/next';
```

Comment:
I wonder if we want this one to actually be a demo.

Reply:
I can see it either way, though lean slightly closer to example because typeahead/multi-typeahead are pretty standard select use cases. 

All these examples use other components (MenuToggle and what goes into the toggle) so it's a bit unclear whether that makes them count more as demos.

Reply:
yeah... I was debating even commenting....

Reply:
Yeah it's a tricky set of examples for sure. I think we can probably leave the basic use cases as examples for now at least, and then when we get into the more complex use cases (like favorites, extra actions, etc) maybe those may be more fitting for demos.